### PR TITLE
Optik manage phase2

### PR DESCRIPTION
**Pull request by jas-lamba**

### CODE REVIEW

File: mytelus-analytics/input/page_views.json [hunk 1]
```diff
@@ -77,22 +77,57 @@
     "plans":{
         "aa.app.section": "plans",
         "overview":{
-            "aa.page.name": "app/my-telus/plans/individual"
+            "aa.page.name": "app/my-telus/plans/individual",
+            "aa.product.status" : "new"
```
Comment (prateem) [inline mytelus-analytics/input/page_views.json]: This is meant to be added to every plans overview page view? (e.g. also for wireless accounts)

If this is meant to be Optik only, consider moving this into the `optik` block

Comment (jas-lamba) [reply mytelus-analytics/input/page_views.json]: addressed in latest push

**Comment by prateem** [comment]
@jas-lamba These are the changes that went into rc 4.15.0 right?

**Comment by jas-lamba** [comment]
@prateem Yes, these are the changes that went into 4.15

**Pull request merged**